extensions/amp-a4a/0.1/signature-verifier.js

```diff
@@ -336,7 +336,7 @@ export class SignatureVerifier {
           );
           return response.json().then(
             jsonResponse => {
-              const jwkSet = /** @type {!JsonObject} */ (jsonResponse);
+              const jwkSet = jsonResponse;
```

Comment:
it isn't. so this is definitely an instance were a cast is needed. I added a way to cast using `dict` by bypassing the `dict-string-keys` lint rule when it sees a `@forcejsonobjectcast` annotation. I did it this way so that its a lot harder to cast to JsonObject than if we just had a normal function helper (like dict) to cast